edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Amqp/linkhandlers/ModuleMessageLinkHandler.cs

```diff
@@ -36,5 +37,30 @@ protected override async Task OnOpenAsync(TimeSpan timeout)
             await base.OnOpenAsync(timeout);
             await this.DeviceListener.AddSubscription(DeviceSubscription.ModuleMessages);
         }
+
+        protected override void OnMessageSent()
+        {
+            Metrics.AddSentMessage(this.Identity);
+        }
+
+        static class Metrics
+        {
+            static readonly IMetricsMeter SentMessagesMeter = Util.Metrics.Metrics.Instance.CreateMeter(
+                "messages_sent",
```

Comment:
`messages_sent_total`?

Reply:
So, if we use a meter, AppMetrics adds _total to the name! :)